deps/rtmidi/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required (VERSION 2.8.11)
+project(rtmidi)
+
+#if (WIN32)
```

Comment:
If this commented out block is not required, remove it.  If it's meant for documentation, make it clear that this is documentation.

Reply:
whoops, that should've been removed I forgot to stage that change.